Python/peephole.c

```diff
@@ -427,11 +447,14 @@ PyCode_Optimize(PyObject *code, PyObject* consts, PyObject *names,
 
     /* Fixup lnotab */
     for (i = 0, nops = 0; i < codelen; i++) {
-        assert(i - nops <= INT_MAX);
+        size_t block = (size_t)i - nops;
```

Comment:
Maybe make `i` an `unsigned int`?

Reply:
> Maybe make i an unsigned int?

I hesitated to do that, but right now the scope of 'i' is the whole function and the function is very long. I tried to write the minimum change.